install/css/install.css

```diff
@@ -191,6 +191,10 @@ input[type="url"] {
 	width: 100%;
 }
 
+input[type="number"] {
```

Comment:
It was probably not good to do that with other inputs but we're stuck with them.